pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -17,21 +17,22 @@ package networkpolicy
 import (
 	"context"
 	"fmt"
-	"math"
 	"net"
 	"os"
 	"regexp"
 	"strings"
 	"sync"
 	"time"
 
-	"antrea.io/libOpenflow/protocol"
-	"antrea.io/ofnet/ofctrl"
 	"github.com/miekg/dns"
 	"k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
+	"k8s.io/utils/clock"
+
+	"antrea.io/libOpenflow/protocol"
+	"antrea.io/ofnet/ofctrl"
```

Comment:
Thanks for the update , i tried runnig the ```make golangci``` command  and i get an error 
``` pkg/agent/controller/networkpolicy/fqdn.go:33: File is not `goimports`-ed with -local antrea.io/antrea (goimports)``` 
for the followng format of imports 
```go
package networkpolicy

import (
	"context"
	"fmt"
	"net"
	"os"
	"regexp"
	"strings"
	"sync"
	"time"

	"github.com/miekg/dns"
	"k8s.io/apimachinery/pkg/util/errors"
	"k8s.io/apimachinery/pkg/util/sets"
	"k8s.io/client-go/util/workqueue"
	"k8s.io/klog/v2"
	"k8s.io/utils/clock"

	"antrea.io/antrea/pkg/agent/openflow"
	"antrea.io/antrea/pkg/agent/types"
	binding "antrea.io/antrea/pkg/ovs/openflow"
	utilsets "antrea.io/antrea/pkg/util/sets"
	dnsutil "antrea.io/antrea/third_party/dns"
	"antrea.io/libOpenflow/protocol"
	"antrea.io/ofnet/ofctrl"
)
```

though they seem to be in order as ```antrea.io/antrea``` imports are in their own section, following all other external imports, but because of the error presented i run ```make golangci-fix``` and it formats the imports as follows :

```go
import (
	"context"
	"fmt"
	"net"
	"os"
	"regexp"
	"strings"
	"sync"
	"time"

	"github.com/miekg/dns"
	"k8s.io/apimachinery/pkg/util/errors"
	"k8s.io/apimachinery/pkg/util/sets"
	"k8s.io/client-go/util/workqueue"
	"k8s.io/klog/v2"
	"k8s.io/utils/clock"

	"antrea.io/libOpenflow/protocol"
	"antrea.io/ofnet/ofctrl"

	"antrea.io/antrea/pkg/agent/openflow"
	"antrea.io/antrea/pkg/agent/types"
	binding "antrea.io/antrea/pkg/ovs/openflow"
	utilsets "antrea.io/antrea/pkg/util/sets"
	dnsutil "antrea.io/antrea/third_party/dns"
)
```

which gets caught as error when running ```make golangci```

Reply:
you are mistaken, Quan didn't mean:
```go
import (
	"context"
	"fmt"
	"net"
	"os"
	"regexp"
	"strings"
	"sync"
	"time"

	"github.com/miekg/dns"
	"k8s.io/apimachinery/pkg/util/errors"
	"k8s.io/apimachinery/pkg/util/sets"
	"k8s.io/client-go/util/workqueue"
	"k8s.io/klog/v2"
	"k8s.io/utils/clock"

	"antrea.io/antrea/pkg/agent/openflow"
	"antrea.io/antrea/pkg/agent/types"
	binding "antrea.io/antrea/pkg/ovs/openflow"
	utilsets "antrea.io/antrea/pkg/util/sets"
	dnsutil "antrea.io/antrea/third_party/dns"
	"antrea.io/libOpenflow/protocol"
	"antrea.io/ofnet/ofctrl"
)
```

He meant:
```go
import (
	"context"
	"fmt"
	"net"
	"os"
	"regexp"
	"strings"
	"sync"
	"time"

	"antrea.io/libOpenflow/protocol"
	"antrea.io/ofnet/ofctrl"
	"github.com/miekg/dns"
	"k8s.io/apimachinery/pkg/util/errors"
	"k8s.io/apimachinery/pkg/util/sets"
	"k8s.io/client-go/util/workqueue"
	"k8s.io/klog/v2"
	"k8s.io/utils/clock"

	"antrea.io/antrea/pkg/agent/openflow"
	"antrea.io/antrea/pkg/agent/types"
	binding "antrea.io/antrea/pkg/ovs/openflow"
	utilsets "antrea.io/antrea/pkg/util/sets"
	dnsutil "antrea.io/antrea/third_party/dns"
)
```

Only `antrea.io/antrea` imports are local. Imports from other `antrea.io` modules (`antrea.io/libOpenflow`, `antrea.io/ofnet`) are considered external and are grouped with other third-party modules (such as the `k8s.io` ones).

Please address